README.rst

```diff
@@ -266,7 +266,7 @@ Red Hat family
 SUSE family
 ~~~~~~~~~~~
 
-- openSUSE Leap 42.2/42.3
+- openSUSE Leap 42.3
```

Comment:
I guess Leap releases `15.0` and `15.1` are missing here, since the script attempts to proceed on them when detected.

Reply:
Yeah, I am working on 15 support today. @noelmcloughlin Made some improvements there already, but I am making it a bit more robust with git support, etc. So I will update the README about that in another PR.